appinventor/components-ios/src/Sprite.swift

```diff
@@ -403,7 +403,30 @@ open class Sprite: ViewComponent, UIGestureRecognizerDelegate {
     updateY(y: y)
     registerChanges()
   }
-  
+
+  /**
+   * Moves the sprite to given point after checking whether the coordinates are numbers.
+   */
+  @objc open func MoveToPoint(_ coordinates: YailList<AnyObject>) {
+    let xCoord = CoerceToDouble(coordinates[1] as AnyObject)
+    let yCoord = CoerceToDouble(coordinates[2] as AnyObject)
+    self.MoveTo(xCoord, yCoord)
+  }
+
+  /**
+   * Helper function for MoveToPoint
+   */
+  func CoerceToDouble(_ coordinate: AnyObject ) -> Double {
```

Comment:
@axz0315 Note that only functions that correspond to blocks (i.e., those annotated with `@objc`) capitalize the first letter. Since this is an internal function it should start with a lower case letter, and you can also mark it `private` as its only needed by the Sprite class.

Reply:
Changed! Is there a way I can update this PR? (Do I need to push to upstream ucr again or?)

Reply:
@axz0315 You can push your changes to your existing branch and the PR will update automatically.